src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,53 @@
+.watch {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 500px;
+  width: 500px;
+  border-radius: 50%;
```

Comment:
you can make it mixin with a dot in center

Reply:
ok